### DISCUSSION TITLE
Request Timeout for the client

### DISCUSSION THREAD

**Discussion by Rahul14189** [gijzelaerr/python-snap7#466] [Q&A] [answered]
Hi, I am looking to set request_timeout for the 'connect' method,   but i don't see any such way. I can see inside 'create' method and 'connect' method , 'Cli_Create()' and 'Cli_ConnectTo' methods are called respectively, and these functions are not in python language. So how can i have request_time here or is there any method that i am missing !!

**Comment by gijzelaerr** [comment, marked as answer]
Closing as stale. In python-snap7 3.0, you can set connection timeout via the `connect()` method parameters.
